Update jet configs; minor doc update

ulmononian opened the pull request:
- Add a `mirror.yaml` for Jet
- Add `libxpm` as an an external in Jet packages.yaml (breaks UE install w/out it).
- Update Jet prereq. load section in Platforms chapter
- Add `spack install --source --verbose` to Maintainers chapter

ulmononian committed:
Add ufs-wm-dev template to develop.

ulmononian committed:
Remove ufs-wm-dev template to get fork dev aligned with main dev.

ulmononian committed:
Merge branch 'NOAA-EMC:develop' into develop

ulmononian committed:
Merge branch 'NOAA-EMC:develop' into develop

ulmononian committed:
Merge branch 'NOAA-EMC:develop' into develop

ulmononian committed:
Merge branch 'NOAA-EMC:develop' into develop

ulmononian committed:
Merge branch 'NOAA-EMC:develop' into develop

ulmononian committed:
Merge branch 'NOAA-EMC:develop' into develop

ulmononian committed:
Update .gitmodules

ulmononian committed:
Update .gitmodules

Revert to EMC spack-stack dev

ulmononian committed:
Merge branch 'NOAA-EMC:develop' into develop

ulmononian committed:
Merge branch 'NOAA-EMC:develop' into develop

ulmononian committed:
Merge branch 'NOAA-EMC:develop' into develop

Commit:
Fix .gitmodules branch in develop of fork.

ulmononian committed:
Merge branch 'NOAA-EMC:develop' into develop

ulmononian committed:
Merge branch 'NOAA-EMC:develop' into develop

Commit:
Add libxpm as external to Jet packages.yaml; add mirror.yaml for Jet; minor doc updates.

climbfuji commented:
This needs to go into the release branch, I'll make the change.

climbfuji commented on doc/source/Platforms.rst:
```diff
@@ -393,8 +393,17 @@ Note that on Hera, a dedicated node exists for ``ecflow`` server jobs (``hecflow
+   module load mysql/8.0.31
+
 
 **WORK IN PROGRESS**
```
Do we still need the work in progress? I guess it's ok and we add the information for loading the spack-stack-1.3.0 unified-env for all platforms at once later.

climbfuji replied:
Same goes for the table at the top of the page.

ulmononian replied:
that is why i left it...will remove when we add the spack-stack-1.3.0 unified-env as you said.

climbfuji commented:
You'll need to resolve the conflicts in `doc/source/Platforms.rst` before it can be merged.

Commit:
Resolve conflicts.

ulmononian commented:
> You'll need to resolve the conflicts in `doc/source/Platforms.rst` before it can be merged.

am i missing something? i updated to the release version of PreconfiguredSites.rst and added the jet change. idk how that is going to be resolved...

climbfuji commented:
I'll fix it tomorrow and either push to your branch our send you a PR to update your branch.

climbfuji committed:
Merge branch 'release/1.3.0' of https://github.com/NOAA-EMC/spack-stack into update_jet

ulmononian commented:
> I'll fix it tomorrow and either push to your branch our send you a PR to update your branch.

thanks for doing that, @climbfuji.
